ground/gcs/src/plugins/opmap/opmapgadgetwidget.cpp

```diff
@@ -265,6 +265,18 @@ OPMapGadgetWidget::OPMapGadgetWidget(QWidget *parent) : QWidget(parent)
         }
     }
 
+    // Update the home location if it is already set:
+    if (uavo_util_mgr) {
+        bool set;
+        double LLA[3];
+        if (uavo_util_mgr->getHomeLocation(set, LLA) == 0) {
+            if (LLA[0] != 0 && LLA[1] != 0 && LLA[2] != 0)
```

Comment:
I thought it always turned back to false after it is set? Ok, probably my mistake, I'll update.